src/FTPC.jl

```diff
@@ -229,7 +229,7 @@ function setup_easy_handle(options::RequestOptions)
     @ce_curl curl_easy_setopt CURLOPT_URL ctxt.url
     # @ce_curl curl_easy_setopt CURLOPT_VERBOSE Int64(1)
 
-    if options.ssl
+    if options.ssl || parse(Bool, get(ENV, "FTPES", "false"))
```

Comment:
I was suggesting that we move away from the ENV variable for this and rather parse this from the protocol given in the URL. Also, this should be evaluated in the constructor and not when we establish the connection.

Reply:
Ah ok I see

Reply:
@omus so "ftp://" would mean no security, "ftpes://" would mean explicit ssl, and "ftps://" implicit ssl?

Reply:
Yes, that's what I was thinking. Does that seem reasonable?

Reply:
Yes. But it might be misleading to pass in "ftpes://" but have to change it to "ftps://" so that libcurl knows what to do with it?

Reply:
We'd have to change "ftpes://" back into "ftp://" so that libcurl can understand it.